libpod/options.go

```diff
@@ -885,6 +885,24 @@ func WithUserVolumes(volumes []string) CtrCreateOption {
 	}
 }
 
+// WithBuiltInVolumesFrom sets the built-in volumes of the container retrieved
+// from a container passed in to the --volumes-from flag.
+// This stores the built-in volume information in the ContainerConfig so we can
+// add them when creating the container.
+func WithBuiltInVolumesFrom(volumes []string) CtrCreateOption {
```

Comment:
I'm thinking we should rename this, as it's not just about image volumes from another container - it's just creating a volume mounted to a local directory owned by the container. Or a tmpfs, if image volumes are set to tmpfs, but that would happen in the podman side. Maybe rename to `WithLocalVolumes()` and change `ctr.config.BuiltInVolumesFrom` to `ctr.config.LocalVolumes`

Reply:
Fixed